forEach.js

```diff
@@ -11,3 +11,35 @@ function cb(item, index, arr){
 };
 
 customForEach(family, cb);
+
+// Slice
+const array = ["t", "e", "s", "t"];
+
+function slice(start, end) {
```

Comment:
Make this function dynamic so you can use it with all arrays

```suggestion
function slice(arr, start, end) {
```